src/pages/docs/tutorials/tutorial-kitten-browser/panels/index.md

```diff
@@ -70,7 +73,12 @@ There are a couple of things to discuss, however. First, we want to add a [`prop
 
 We have a small problem with our `Detail` view. We don't validate that the gender we receive matches one of the genders we expect. One way we can address that is to use `propTypes` to validate that we only receive the data we expect (at least, while we're running the app in development mode). We can quickly change the validator to check the data for us:
 
-	gender: PropTypes.oneOf(['m', 'f']),
+	propTypes: {
+			color: PropTypes.string,
+			gender: PropTypes.oneOf(['m', 'f']),
+			name: PropTypes.string,
+			weight: PropTypes.number
+	},
```

Comment:
I think this is not needed here.

Reply:
I reverted it to original description. So there will be only "oneOf" description.